WordPress/Classes/Networking/ReaderPostServiceRemote.m

```diff
@@ -9,6 +9,8 @@
 #import "WordPress-Swift.h"
 #import "PhotonImageURLHelper.h"
 
+CGFloat const ReaderPostServiceRemotOffsetRankCap = 1000000.0;
```

Comment:
_Maybe_ a comment here for why this number is particularly set to `1000000.0`, although it does say below that it is an arbitrarily high number.